### ISSUE TITLE
2019Q4 - 1.5 Outreach Backlog Burndown

### ISSUE THREAD

**Issue by miketaylr** [mozilla/webcompat-team-okrs#102]
Let's get in touch with 200 sites in the next 12 weeks. That's a lot! What's the best way to track this work?

**Comment by karlcow** [comment]
Status: 
* 11 [needscontact - priority critical](https://github.com/webcompat/web-bugs/issues?q=is%3Aopen+is%3Aissue+milestone%3Aneedscontact+label%3Apriority-critical)
* 16 [contactready - priority critical](https://github.com/webcompat/web-bugs/issues?q=is%3Aopen+is%3Aissue+label%3Apriority-critical+milestone%3Acontactready) (Total: 417)

**Comment by karlcow** [comment]
Contacts made on 2019-10-03
* https://github.com/webcompat/web-bugs/issues/31923
* https://github.com/webcompat/web-bugs/issues/31685
* https://github.com/webcompat/web-bugs/issues/31619
* https://github.com/webcompat/web-bugs/issues/4070
* https://github.com/webcompat/web-bugs/issues/37976

**Comment by miketaylr** [comment]
> 11 [needscontact - priority critical](https://github.com/webcompat/web-bugs/issues?q=is%3Aopen+is%3Aissue+milestone%3Aneedscontact+label%3Apriority-critical)

I was able to close most of these (all but 1) as fixed, dupes, or move to sitewait.

**Comment by karlcow** [comment]
Contacts made 2019-10-07

* https://github.com/webcompat/web-bugs/issues/14172#issuecomment-538806598
* https://github.com/webcompat/web-bugs/issues/33963
* https://github.com/webcompat/web-bugs/issues/32592
* https://github.com/webcompat/web-bugs/issues/25324
* https://github.com/webcompat/web-bugs/issues/19122

needscontact: 408
contactready: 83

**Comment by karlcow** [comment]
Contacts made 2019-10-08

* https://github.com/webcompat/web-bugs/issues/8616
* https://github.com/webcompat/web-bugs/issues/31591
* https://github.com/webcompat/web-bugs/issues/27429
* https://github.com/webcompat/web-bugs/issues/22990


needscontact: 387
contactready: 83

**Comment by haseebgit** [comment]
Contacts made on 2019-10-09

- https://github.com/webcompat/web-bugs/issues/34948
- https://github.com/webcompat/web-bugs/issues/39363
- https://github.com/webcompat/web-bugs/issues/39792
- https://github.com/webcompat/web-bugs/issues/39826
- https://github.com/webcompat/web-bugs/issues/26077

needscontact: 389
contactready: 83

**Comment by karlcow** [comment]
Contacts made on 2019-10-14

Contacted 3/4 sites. And closed issues which could not be reproduced. 
Also asked in friends in Mozilla Beijing can help us for Chinese Websites.

needscontact: 377
contactready: 88

**Comment by karlcow** [comment]
Contacts made on 2019-10-15

needscontact: 367 (0 critical, 37 important)
contactready: 88 (7 critical, 34 important)

**Comment by karlcow** [comment]
There is a bit of progress here. Maybe less than previous weeks.
Stats on today 2019-10-29

needscontact: 344 (0 critical, 40 important)
contactready: 85 (2 critical, 34 important)

**Comment by miketaylr** [comment]
Update: assuming everyone is labeling their issues, we have 34 out of 200 currently.

**Comment by laghee** [comment]
I added the label to some of the above referenced issues that didn't have it, and the count is now at 53.

Wasn't sure if these two closed issues should count, but they're included for now:

- https://github.com/webcompat/web-bugs/issues/33963
- https://github.com/webcompat/web-bugs/issues/19122

**Comment by miketaylr** [comment]
Currently at 63/200.

**Comment by karlcow** [comment]
68/200
https://github.com/webcompat/web-bugs/issues?utf8=%E2%9C%93&q=+is%3Aissue+label%3Aq4-2019-outreach+

This is slow, because I also close many as invalid or fixed :) a ratio maybe of 1 to 4.

**Comment by karlcow** [comment]
currently [78/200](https://github.com/webcompat/web-bugs/issues?utf8=%E2%9C%93&q=+is%3Aissue+label%3Aq4-2019-outreach+)

**Comment by karlcow** [comment]
![Capture d’écran 2019-12-12 à 16 36 56](https://user-images.githubusercontent.com/505230/70691944-acd27a00-1cfd-11ea-93e7-3a0e86b49793.png)
![Capture d’écran 2019-12-12 à 16 36 25](https://user-images.githubusercontent.com/505230/70691949-b0fe9780-1cfd-11ea-9dd2-a177171de4e6.png)

**Comment by karlcow** [comment]
Currently [87/200](https://github.com/webcompat/web-bugs/issues?utf8=%E2%9C%93&q=+is%3Aissue+label%3Aq4-2019-outreach+)

**Comment by karlcow** [comment]
I made a bit of progress on this the last couple of days. But I don't think we will reach the target before the end of the year. Note that in the process we closed bugs. :)

Approximate number (needscontact + contact ready)
Beginning of quarter: 335 + 90 = 425
Current: 290 + 31 = 321

That's a differential of ~100 (it doesn't mean 100 issues have been addressed, probably a lot more than this because we do not count the in-flow and out-flow for each milestone.)
so we are in a better position than before.
